src/System.Runtime.Extensions/tests/System/Convert.netcoreapp.cs

```diff
@@ -157,5 +148,29 @@ public void TryFromBase64Chars_MatchesFromBase64CharArray(string stringInput)
             Assert.Equal<byte>(expected, dest.Slice(0, dest.Length - 1).ToArray());
             Assert.Equal(0, dest[dest.Length - 1]);
         }
+
+        public static IEnumerable<object[]> FromBase64TestData
```

Comment:
It might be worth grabbing the 1-2 dozen base64 test cases directly from https://tools.ietf.org/html/rfc4648#section-10